### DISCUSSION TITLE
ClamAV crashing

### DISCUSSION THREAD

**Discussion by MaoMaoCake** [nextcloud/all-in-one#5295] [Questions] [answered]
Issue:
ClamAV container turns unresponsive after 24hrs

From the following discussion:
https://github.com/nextcloud/all-in-one/discussions/4477
and the relavant upstream issue:
https://github.com/Cisco-Talos/clamav-docker/issues/45

we know that clamAV-docker currently has issues with reliability. This is a bug report / feature request since we already know its a bug from upstream. 

Possible solutions:
Roll back to aio-clamav:adaed56c7deb from about 4 months ago to restore functionality

**Comment by szaimen** [comment]
Hi @MaoMaoCake thanks for the report. I fear we run into a dilemma with this now. I we roll back to that image, we probably introduce security issues. However if we don't, the image is unreliable. The best solution of course would be if upstream would finally fix the issue...

**Comment by MaoMaoCake** [comment]
would it be possible for a user initiated downgrade or is that out of the supported scope of AIO?

**Comment by szaimen** [comment]
You could install the version from back then but it will downgrade all components not only the clamav container

**Comment by MaoMaoCake** [comment]
I feel like that's a worse solution than downgrading just clamav.
I think I'll spin up a VM dedicated to clamav in the time being and set AIO to use that instead. Hopefully upstream solves the issue soon.

**Comment by szaimen** [comment]
> I think I'll spin up a VM dedicated to clamav in the time being and set AIO to use that instead.

Sounds likke a good workaround. You will need this: https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-keep-disabled-apps

**Comment by szaimen** [comment]
I'll move this to discussions for the time being as this is currently not actionable feom the aio side.

**Comment by szaimen** [comment, marked as answer]
Hi, this should have been resolved when we migrated to our own clamav build with AIO v10.8.0. See https://github.com/nextcloud/all-in-one/pull/6094
